apps/project/tests/test_apis.py

```diff
@@ -1402,38 +1402,97 @@ def test_project_summary_api(self):
-        self.update_obj(self.create_entry(lead=lead3, verified=True), created_at=now + relativedelta(days=-10))
-        self.update_obj(self.create_entry(lead=lead3, verified=True), created_at=now + relativedelta(days=-20))
-        self.update_obj(self.create_entry(lead=lead3, verified=True), created_at=now + relativedelta(days=-30))
-        self.update_obj(self.create_entry(lead=lead3, verified=True), created_at=now + relativedelta(days=-40))
-        self.update_obj(self.create_entry(lead=lead4, verified=False), created_at=now + relativedelta(months=-3, days=-1))
+        self.update_obj(
+            self.create_entry(lead=lead1, verified=True, created_by=user),
+            created_at=now + relativedelta(months=-3, days=-1)
+        )
+        self.update_obj(
+            self.create_entry(lead=lead1, verified=True, created_by=user),
+            created_at=now + relativedelta(months=-2)
+        )
+        self.update_obj(
+            self.create_entry(lead=lead2, verified=False, created_by=user),
+            created_at=now + relativedelta(months=-3)
+        )
+        self.update_obj(
+            self.create_entry(lead=lead2, verified=True, created_by=user),
+            created_at=now + relativedelta(months=-3)
+        )
+        self.update_obj(
+            self.create_entry(lead=lead2, verified=True, created_by=user),
+            created_at=now + relativedelta(months=-3)
+        )
+        self.update_obj(
+            self.create_entry(lead=lead3, verified=True, created_by=user),
+            created_at=now + relativedelta(days=-10)
+        )
+        self.update_obj(
+            self.create_entry(lead=lead3, verified=True, created_by=user),
+            created_at=now + relativedelta(days=-20)
+        )
+        self.update_obj(
+            self.create_entry(lead=lead3, verified=True, created_by=user),
+            created_at=now + relativedelta(days=-30)
+        )
+        self.update_obj(
+            self.create_entry(lead=lead3, verified=True, created_by=user),
+            created_at=now + relativedelta(days=-40)
+        )
+        self.update_obj(
+            self.create_entry(lead=lead4, verified=False, created_by=user),
+            created_at=now + relativedelta(months=-3, days=-1)
+        )
+        self.update_obj(
+            self.create_entry(lead=lead5, verified=True, created_by=user),
+            created_at=now + relativedelta(months=-3)
+        )
+        self.update_obj(
+            self.create_entry(lead=lead5, verified=True, created_by=user),
+            created_at=now + relativedelta(months=-2)
+        )
+        self.update_obj(
+            self.create_entry(lead=lead6, verified=True, created_by=user),
+            created_at=now + relativedelta(months=-3, days=-1)
+        )
+        self.update_obj(
+            self.create_entry(lead=lead7, verified=True, created_by=user),
+            created_at=now + relativedelta(months=-3)
+        )
```

Comment:
This approach will be more cleaner

data = [{"lead": lead1, "verifield": True, "months":1}, {"lead": lead2, "verifield": True, "months": 2}, ...]
for item in data:
    self.update_obj(
            self.create_entry(lead=item["lead"], verified=item["verifield", created_by=user),
            created_at=now + relativedelta(months=item["months"])
        )